tool/tsh/common/kube.go

```diff
@@ -1588,3 +1666,24 @@ func accessRequestForKubeCluster(ctx context.Context, cf *CLIConf, tc *client.Te
 
 	return req, nil
 }
+
+// formatAmbiguousKubeCluster is a helper func that formats an ambiguous kube
+// cluster error message.
+func formatAmbiguousKubeCluster(cf *CLIConf, selectors resourceSelectors, kubeClusters types.KubeClusters) string {
+	// dont mark the selected cluster
```

Comment:
```suggestion
	// don't mark the selected cluster
```

Reply:
> But wonder if we can delay backporting to v13 resource name collision CLI changes after we will finish v13 test plan regression.

I agree, and I've been delaying backporting these UX changes until the testplan so we avoid backporting regressions like what happened with my `tsh db` ux backport a little while ago

Reply:
>Additionally we have also the tsh proxy kube command that needs to be extended.

Separate PR will be coming for that, thanks for pointing it out